### PR TITLE
Fix flake8 noqa: `_fuzzy_join`

### PR DESCRIPTION
**Pull request by juanitorduz**
Partially closes https://github.com/skrub-data/skrub/issues/755

See https://github.com/skrub-data/skrub/pull/765

### CODE REVIEW

File: skrub/_fuzzy_join.py [hunk 1]
```diff
@@ -208,7 +205,8 @@ def fuzzy_join(
     sort: bool = False,
     suffixes: tuple[str, str] = ("_x", "_y"),
 ) -> pd.DataFrame:
-    """Join two tables based on approximate matching using the appropriate similarity metric.
+    """Join two tables based on approximate matching using the appropriate similarity
```
Comment (Vincent-Maladiere) [inline skrub/_fuzzy_join.py:208]: Ditto here:
```suggestion
    """Join two tables based on approximate matching using the appropriate similarity \
```

**Review by Vincent-Maladiere — COMMENTED** [review]
You also have an error here because of the single-line description.

**Comment by juanitorduz** [comment]
ok! thanks for the input! I will make the changes if needed 👍

**Review by Vincent-Maladiere — APPROVED** [review]
LGTM, thanks!

**Pull request merged**